extensions/wikia/CreatePage/CreatePage.php

```diff
@@ -89,7 +89,9 @@ function wfCreatePageSetupVars(Array &$vars ) {
 		$contentNamespaces[] = $wgContLang->getNsText( $contentNs );
 	}
 
-	$vars['WikiaEnableNewCreatepage'] = $wgUser->getOption( 'createpagepopupdisabled', false ) ? false : $wgWikiaEnableNewCreatepageExt;
+	if ( !isset( $vars['WikiaEnableNewCreatepage'] ) ) {
```

Comment:
Line of motivation would be welcomed here to be more explicit why we want to omit setting this property